test/function_tests/backplane_operator_install_test/backplane_operator_install_test.go

```diff
@@ -198,6 +203,58 @@ var _ = Describe("BackplaneConfig Test Suite", func() {
 				})
 			}
 		})
+		It("Should check that the config spec has propagated", func() {
+
+			tests := []struct {
```

Comment:
Rather than hard coding the deployment names it would be better to get a list of all deployments we create since that could change over time. We set Backplane labels on the deployments we stand up so they could found via labels, and then iterated through that list. Check out https://github.com/open-cluster-management/discovery/blob/main/test/e2e/discoveryconfig_controller.go#L570 for an example of how to do this.